integration/mediation-tests/tests-mediator-1/src/test/java/org/wso2/carbon/esb/message/processor/test/DeactivatedCappMPBehaviourOnRestartTestCase.java

```diff
@@ -80,15 +71,13 @@ public void testDeactivatedMPUponServerRestart() throws Exception {
         axis2Server.start();
 
         // Restart the ESB Server after the MP is deactivated.
-        serverConfigurationManager.restartGracefully();
+        CarbonServerExtension.restartServer();
 
         super.init();
-        logViewerClient = new LogViewerClient(contextUrls.getBackEndUrl(), getSessionCookie());
 
-        Assert.assertFalse(Utils.checkForLog(logViewerClient, EXPECTED_ERROR_MESSAGE, 10),
+        Assert.assertFalse(Utils.logExists(carbonLogReader, EXPECTED_ERROR_MESSAGE, 10),
                 "Error occurred while deploying Carbon Application!");
-        Assert.assertTrue(Utils.checkForLog(logViewerClient, SUCCESS_MESSAGE, 10),
-                "Unable to deploy Carbon Application!");
+        Assert.assertFalse(Utils.logExists(carbonLogReader, SUCCESS_MESSAGE, 10));
```

Comment:
I guess this was assert true . Why we have changed as assert false

Reply:
Is it a copy paste mistake?